libs/core/src/components/sage-divider/sage-divider.scss

```diff
@@ -0,0 +1,46 @@
+/*
+TODO: update offset values to use
+our spacing tokens when available.
+*/
+
+$sage-divider-offsets: (
+  xxs: 4px,
+  xs: 8px,
+  sm: 16px,
+  md: 24px,
+  lg: 32px,
+  xl: 48px,
+  xxl: 64px,
+);
+
+.sage-divider {
+  background: #D3D5D9;
+  border: 0;
+  display: flex;
+  height: 1px;
+  margin: 0;
+  width: 100%;
+
+  &--vertical {
```

Comment:
I'm not the biggest fan of concatenating class names when it can be easily avoided as it makes searching/find and replacing more challenging. I wanted to voice it but ultimately its your call here 👍

Reply:
Updated.